hub-templates/basehub/values.yaml

```diff
@@ -29,10 +29,15 @@ nfsPVC:
 
 jupyterhub:
   custom:
+    singleuserAdmin:
+      extraVolumeMounts:
+        - name: home
+          mountPath: /home/jovyan/shared-readwrite
+          subPath: _shared
     cloudResources:
-      provider: null
+      provider:
```

Comment:
Is there any difference between leaving this black and not setting them at all? Or is this a way to unset default values?

Reply:
ruamel.yaml formatted it this way, I assume it's equivalent?